packages/react-composites/review/react-composites.api.md

```diff
@@ -520,6 +520,9 @@ export interface ChatCompositeStrings {
     chatListHeader: string;
 }
 
+// @beta
+export const completedFileUpload: (data: FileMetadata) => ObservableFileUpload;
```

Comment:
I'd name it `createCompletedFileUpload` or (golang-convetion) `newCompletedFileUpload` to indicate that this is creating a new instance of a completed upload.